torch_geometric/transforms/constant.py

```diff
@@ -14,15 +14,28 @@ class Constant(BaseTransform):
 
     Args:
         value (float, optional): The value to add. (default: :obj:`1.0`)
-        cat (bool, optional): If set to :obj:`False`, all existing node
+        cat (bool, optional): If set to :obj:`False`, the node
             features will be replaced. (default: :obj:`True`)
+        node_types (str or List[str], optional): The specified node types to
+            add constants, which is used only for heterogeneous graphs.
+            If set to :obj:`None`, constants will be added to each node feature
+            :obj:`x` of all existing node types. (default: :obj:`None`)
     """
-    def __init__(self, value: float = 1.0, cat: bool = True):
+    def __init__(self, value: float = 1.0, cat: bool = True,
+                 node_types: Optional[Union[str, List[str]]] = None):
         self.value = value
         self.cat = cat
-
-    def __call__(self, data: Union[Data, HeteroData]):
-        for store in data.node_stores:
+        self.node_types = node_types
+
+    def __call__(self, data: Union[Data,
+                                   HeteroData]) -> Union[Data, HeteroData]:
+        if isinstance(data, HeteroData) and self.node_types is not None:
```

Comment:
should we just assert `self.node_types` is not None if using HeteroData?

Reply:
Constants will be added to all node features if  `self.node_types=None`. This is to be consistent with the previous implementation where constants are added to all existing features by default. Or perhaps we should assert `self.node_types` is None if using Data?

Reply:
okay makes sense 👍